easybuild/easyconfigs/p/pugixml/pugixml-1.12.1-GCCcore-11.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'pugixml'
+version = '1.12.1'
+
+homepage = 'https://pugixml.org/'
+description = """pugixml is a light-weight C++ XML processing library"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/zeux/pugixml/archive/']
+sources = ['v%(version)s.tar.gz']
+
+checksums = [
+    '1e28ab24b6e04e013d96f45d25e9f2d04c921dc68c613fd010ecaaad3892c14d',  # v1.12.1.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.22.1')
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON -DPUGIXML_BUILD_SHARED_AND_STATIC_LIBS=ON -DPUGIXML_HEADER_ONLY=True '
```

The older easyconfig already contains this, but I don't think this `-DPUGIXML_HEADER_ONLY=True` is actually doing anything. In my build log I see:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_C_COMPILER
    CMAKE_C_FLAGS
    CMAKE_FIND_USE_PACKAGE_REGISTRY
    CMAKE_Fortran_COMPILER
    CMAKE_Fortran_FLAGS
    PUGIXML_HEADER_ONLY
```
And I don't see any difference when building with/without this option. The documentation says you have to define this `PUGIXML_HEADER_ONLY` in your " pugiconfig.hpp, or provide them via compiler command-line", so it's not actually a CMake variable.

Hence, I guess we could/should just remove this option here?

```suggestion
configopts = '-DBUILD_SHARED_LIBS=ON -DPUGIXML_BUILD_SHARED_AND_STATIC_LIBS=ON '
```

---

Yes, you are right. I have changed that. Thanks for pointing that out, much appreciated.